python/lsst/pipe/tasks/skyCorrection.py

```diff
@@ -387,6 +400,37 @@ def _restoreBackgroundRefineMask(self, calExps, calBkgs):
             skyCorrBases.append(skyCorrBase)
         return calExps, skyCorrBases
 
+    def _restoreVisitBackground(self, calExps, calBkgs):
```

Comment:
I would suggest renaming this to `_restoreInitialBackground`.

In tandem with this, the current method `_restoreBackgroundRefineMask` should probably also be renamed to `_restoreOriginalBackgroundRefineMask`, to avoid any potential confusion.

Reply:
PS - if we're changing the associated config to `undoBgModel1`, it probably also makes sense to rename this method `_undoInitialBackground` as well.